### PR TITLE
Character talent tab translations

### PR DESCRIPTION
**Pull request by liaojh1998**
## Describe your changes

* Resolves translation comments in #1358.
* Standardized translations to languages in Genshin by the following:
  * Changes "Normal/Charged Attack" to just "Normal Attack". All translations use "Normal Attack".
  * All "Constellation Lv." follows the same translations as Genshin.
  * All "Talent Lv." follows the same translations as Genshin.
  * Instead of "Unlocked at Ascension #", use the same translations in Genshin for "Unlocks at Character Ascension Phase #".
  * Instead of "Unlocked by Default", modified to "Passive Talent" and used the same Genshin translations for "Passive Talent".

## Testing/validation

image
image
image
image

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [x] I have commented my code, in hard-to understand areas.
- [x] I have made corresponding changes to README or wiki.
- [x] For front-end changes, I have updated the corresponding English translations.
- [x] Ran `yarn run mini-ci` locally to validate format + lint.
- [x] If there were format issues, I ran `nx format write` to resolve them automatically.

### CODE REVIEW

**Comment by liaojh1998** [comment]
Blocking until #1358 is submitted.

File: libs/gi-localization/assets/locales/en/page_character.json
Comment (frzyc) [inline libs/gi-localization/assets/locales/en/page_character.json:63]: You can use https://www.i18next.com/translation-function/interpolation to minimize the churn in `constellationLvl` and in `talentLvl`

**Review by frzyc — CHANGES_REQUESTED** [review]
Hey @liaojh1998, I appreciate your work here on the translations. We are currently putting this site on maintenance mode, in favor of working on a new UI with a new calculation engine. so I am generally not scoping time for translations for this UI. However, I want to make sure that your time is respected, and I will work with you to put this translations in, especially since you've spent the time to update the translations.

**Comment by liaojh1998** [comment]
Thanks, I'll hold off on any translation stuff in other changes. Sorry for the inconvenience here.

**Review by PolomoPT — CHANGES_REQUESTED** [review]
I take it those translations were datamined, yeah? You should have them out of the localizations file, then.

Get the string ID and list it under libs/gi-dm-localization/src/executors/gen-locale/lib/Data/sheet.ts, and then reference that instead. We reference stuff in that file typically with `stg`, you can find it in most character pages.

TL;DR: gi-localization is for things that need manual translation, such as conditionals. these extracted strings should be in gi-dm-localization, including the English ones.

**Review by frzyc — APPROVED** [review]

**Review by nguyentvan7 — APPROVED** [review]
Deployed and tested with onetime deployment pipeline. Will clean after merge

**Pull request merged**
